src/rcore.c

```diff
@@ -631,7 +631,12 @@ void InitWindow(int width, int height, const char *title)
 
     // Initialize platform
     //--------------------------------------------------------------
-    InitPlatform();
+    if ( InitPlatform() != 0 )
```

Comment:
I try to avoid this kind of structure on raylib, also, not following conventions, raylib does not place spaces between parenthesis. In any case it should be:
```c
int result = InitPlatform();
if (result == -1)  // Platform initialization failed
{
    TRACELOG(LOG_WARNING, "WINDOW: Platform initialization failed");
    //CORE.Window.ready = false;  // If platform has not been correctly initialized, this should be already "false"
    return;  // In araylib I try to avoid/minimize early returns
}
```